myFirstWebsite/index.html

```diff
@@ -0,0 +1,440 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My First Website</title>
+    <link rel="stylesheet" href="./css/reset.css">
+    <link rel="stylesheet" href="./css/fonts.css">
+    <link rel="stylesheet" href="./css/styles.css">
+    <link rel="shortcut icon" href="./images/site-icon-h179.png" type="image/png">
+</head>
+<body>
+    <div class="header__image-holder">
+        <header class="header">
+            <div class="wrapper">
+                <div class="header__wrapper">
+                    <div class="header__logo">
```

Comment:
It's a good practice to make a logo as a link to the main page, for SPA it usually scrolls to the top of the page